src/Controller/Api/ApiController.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+namespace App\Controller\Api;
+
+use Symfony\Component\HttpFoundation\{Request, Response};
+use Symfony\Component\HttpKernel\Exception\{NotFoundHttpException, UnsupportedMediaTypeHttpException};
+use Symfony\Component\Routing\Annotation\Route;
+
+class ApiController
+{
+    /**
+     * Returns information about client and writes it to log file.
+     *
+     * @Route("/api/my-info", methods="GET", name="client_info")
+     *
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function actionClientInfo(Request $request): Response
+    {
+        if (!$request->headers->get('Content-type: application/json')) {
```

Comment:
Have you tested this check? it should not work.
Also, it make sense to move it to a separate private method, in order to reuse in all the controller actions and reduce the copy-paste.

Reply:
But it works. This check is very simple. So i can't realize why it wouldnt work. 
Yeah, missed this point.